src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="$(MicrosoftVisualStudioLanguageIntellisenseVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime" Version="$(MicrosoftVisualStudioImagingInterop140DesignTimeVersion)" />
     <PackageReference Include="Microsoft.VisualStudio.Text.UI" Version="$(MicrosoftVisualStudioTextUIVersion)" />
+    <PackageReference Include="Microsoft.VisualStudio.Threading" Version="$(MicrosoftVisualStudioThreadingVersion)" />
```

Comment:
Why adding this? Does this need nuspec updates as well?

Reply:
Let us revert this if they are not needed.

---
In reply to: [192217464](https://github.com/dotnet/roslyn/pull/27027#discussion_r192217464) [](ancestors = 192217464)